test/acceptance/features/steps/kubernetesapi.py

```diff
@@ -149,6 +160,7 @@ def on_primaza_cluster_check_service_catalog_augmented(context, cluster, catalog
         timeout=20)
 
 
+@step(u'On Primaza Cluster "{cluster}", ServiceCatalog "{catalog_name}" does not contain RegisteredService "{rs_name}"')
 @then(u'On Primaza Cluster "{cluster}", ServiceCatalog "{catalog_name}" will not contain RegisteredService "{rs_name}"')
```

Comment:
These two steps are the same right, why not combine and change in other feature files

Reply:
Sure